extra-tests/classes/RollupFullRecalcTests.cls

```diff
@@ -655,14 +657,14 @@ private class RollupFullRecalcTests {
 
   @IsTest
   static void clearsParentValuesWhenNoRefreshMatches() {
-    List<Account> accs = [SELECT Id, AnnualRevenue, Description FROM Account];
-    accs[0].AnnualRevenue = 1500;
-    accs[0].Description = 'This is the value that should be persisted';
-    update accs;
+    Account acc = (Account) RollupTestUtils.queryrecord('Account', new List<String>{ 'AnnualRevenue', 'Description' });
```

Comment:
This is fine as-is, but you know me - I always advocate for using field tokens instead of hardcoded strings. Not a big deal though if you want to stick with strings, and I think you're already using strings throughout most of the codebase, but just thought I'd mention it.

```suggestion
    Account acc = (Account) RollupTestUtils.queryrecord('Account', new List<String>{ Account.AnnualRevenue.getDescribe().getName(), Account.Description.getDescribe().getName() });
```

Reply:
let me just check if there were any parent-level field references being passed to this function, otherwise I can easily change it to take in `Schema.SObjectField` tokens instead. the vast majority of the codebase is already using those field tokens - it just may not be obvious as such since they're transformed into String collections by the time they hit the actual production query builder

Reply:
Oh, I was just thinking you could use `SObjectField` tokens to get the string, instead of hardcoding the string. But changing the method to accept parameter of `SObjectField` would be cool too